### PR TITLE
fix: add container_name to docker-compose

### PR DESCRIPTION
**Pull request by akash-jain-10**
### Describe your changes :

Add container_name attribute to docker-compose files. Fix typo in cmd.py

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: docker/local-metadata/docker-compose.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ services:
     build:
       context: ../../.
       dockerfile: docker/local-metadata/Dockerfile_mysql
+    container_name: mysql
```
Comment (harshach) [inline docker/local-metadata/docker-compose.yml:18]: lets call this openmetadata_mysql

File: docker/local-metadata/docker-compose.yml [hunk 2]
```diff
@@ -26,6 +27,7 @@ services:
 
   elasticsearch:
     image: docker.elastic.co/elasticsearch/elasticsearch:7.10.2
+    container_name: elasticsearch
```
Comment (harshach) [inline docker/local-metadata/docker-compose.yml:30]: same here

Comment (akash-jain-10) [reply docker/local-metadata/docker-compose.yml]: Done

File: docker/local-metadata/docker-compose.yml (hunk 1 above)
Comment (akash-jain-10) [reply docker/local-metadata/docker-compose.yml]: Done

**Pull request merged**